### PR TITLE
component tree database design added to readme

### PR DESCRIPTION
**Pull request by debbiect246**
added component tree and database design table to readme - table will need some more work tomorrow as it is not displaying as it should.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ Users can then arrange to buy and collect produce and leave reviews on the produ
 
 
 ## Goal
-Technology has meant that people can now access food online in a variety of apps.  There is also a need for small local producers to have a space where they can showcase the produce they have so that people can purchase it.  
+Technology has meant that people can now access food online in a variety of apps.  There is also a need for small local producers to have a space where they can showcase the produce they have so that people can purchase it.  We worked hard to come up with a project idea during our first sprint and finally we decided on teh idea of this app.
```
Comment (ValeSer) [inline README.md:7]: Misspelled error: the

**Comment by ValeSer** [comment]
Hello :) 
About the technologies and libraries used, you may refer to the package.json both Frontend and Backend (they have their own one) ;)

File: README.md [hunk 2]
```diff
@@ -18,27 +18,55 @@ In the tech section, we provide information about the technology stack, dependen
...
+
 
  
 
 ### Dependencies including deployment
+- The app was deployed to [vercel](https://vercel.com/) 
 
 
 ### Testing
+- testing was done at the start for the project using [jest](https://jestjs.io/)
```
Comment (ValeSer) [inline README.md:69]: I would delete "at the start of the project" since we did not really do that lol

**Review by ValeSer — COMMENTED** [review]
Thank you Debbie. I see there are conflicts to solve, though.

**Comment by bjentwistle** [comment]
I have checked the document and added a link to Express JS and changed a typo - ready for review. Good job Debbie - we can help you add more as we go along this week.

**Review by ValeSer — APPROVED** [review]
ok :)

**Pull request merged**